apps/meteor/tests/end-to-end/api/24-methods.js

```diff
@@ -2397,4 +2397,274 @@ describe('Meteor.methods', function () {
 				});
 		});
 	});
+
+	describe('[@muteUserInRoom & @unmuteUserInRoom]', () => {
+		let rid = null;
+		let channelName = null;
+		let testUser = null;
+		const testUserCredentials = {};
+
+		before('create channel', (done) => {
+			channelName = `methods-test-channel-${Date.now()}`;
+			request
+				.post(api('channels.create'))
+				.set(credentials)
+				.send({
+					name: channelName,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res) => {
+					expect(res.body).to.have.property('success', true);
+					expect(res.body).to.have.nested.property('channel._id');
+					expect(res.body).to.have.nested.property('channel.name', channelName);
+					expect(res.body).to.have.nested.property('channel.t', 'c');
+					expect(res.body).to.have.nested.property('channel.msgs', 0);
+					rid = res.body.channel._id;
+				})
+				.end(done);
+		});
+
+		before('create test user', (done) => {
+			const username = `user.test.${Date.now()}`;
+			const email = `${username}@rocket.chat`;
+			request
+				.post(api('users.create'))
+				.set(credentials)
+				.send({ email, name: username, username, password: username })
+				.end((err, res) => {
+					testUser = res.body.user;
+					done();
+				});
+		});
+
+		before('add user to room', (done) => {
+			request
+				.post(api('channels.invite'))
+				.set(credentials)
+				.send({
+					roomId: rid,
+					userId: testUser._id,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.end(done);
```

Comment:
Provide the `userId` in the `members` array from the `createRoom` helper so that this step isn't needed